shared/desktop/app/menu-helper.desktop.js

```diff
@@ -132,6 +128,8 @@ export default function makeMenu(window: any) {
     ]
     // $FlowIssue not sure yet
     const menu = SafeElectron.Menu.buildFromTemplate(template)
+    window.setAutoHideMenuBar(true)
```

Comment:
this is a bugfix, this was in the wrong location before. but it didn't matter since `frame: false` hides it anyway.